pkg/universe.dagger.io/x/contact@kjuulh.io/rust/image.cue

```diff
@@ -0,0 +1,20 @@
+package rust
+
+import (
+	"universe.dagger.io/docker"
+)
+
+// rust image default version
+_#DefaultVersion: "1.60"
+
+// Pull rust base image
+#Image: {
+	version: *_#DefaultVersion | string
+	docker.#Build & {
+		steps: [
+			docker.#Pull & {
+				source: "rust:\(version)-alpine"
+			},
+		]
+	}
+}
```

Comment:
Done. Copies stuff from go.

Question: Is there a commons directory for universe packages, i.e. if a lot of packages have similar requirements, it may be detrimental to copy code between the packages. Such as packages for an alpine image?

It may also be because `#Image` in this case is a somewhat false abstraction around docker.#Build. I don't really know, and doesn't have enough experience in `cue` to tell, but interesting nonetheless.

Reply:
> Question: Is there a commons directory for universe packages, i.e. if a lot of packages have similar requirements, 

There are no common standard for now, but we are thinking about it :D 
That's basically why I pick that kind of comment.

The purpose of `#Image` is to provide a ready to use image for the package, I agree with you that it's a lot of code duplication and we should certainly find a way to improve that one thanks embedding!